examples/Viewer.tsx

```diff
@@ -311,6 +312,16 @@ class Viewer extends React.Component<{}, ViewerState> {
         });
     }
 
+    public convertFile(obj: Record<string, any>, fileType: string) {
```

Comment:
we should add a specific type for obj here

Reply:
To clarify, if obj is known to consist of form data plus file blob, then it makes sense to have a type for that.